config/processStar.py

```diff
@@ -0,0 +1,19 @@
+import os.path
+from lsst.utils import getPackageDir
+
+config.load(os.path.join(getPackageDir("obs_lsst"), "config", "auxTel", "auxTel.py"))
+
+# Configuration for processStarTask
+
+if hasattr(config, 'ccdKeys'):
```

Comment:
Why do you need this `hasattr` here? When would the config not have that?

Reply:
Agreed. This was cargo-culted and can be removed.